en/firefox/features/translate.ftl

```diff
@@ -4,54 +4,23 @@
-
-# links to https://addons.mozilla.org/firefox/addon/firefox-translations/
-features-translate-get-firefox-translations = Get { -brand-name-firefox-translations }
-
-# "To Google Translate" is a brand name
-features-translate-to-google-translate = To { -brand-name-google } Translate
-# $attrs (string) - link to https://addons.mozilla.org/firefox/addon/to-google-translate/
-features-translate-google-translate-with = { -brand-name-google } Translate, with over 100 languages* at the ready, is used by millions of people around the world. But moving back and forth between translate.google.com and the page you’re trying to read isn’t an ideal experience. The <a { $attrs }>To { -brand-name-google } Translate</a> extension makes translating the page you’re on easier than ever.
-
-# links to: https://addons.mozilla.org/firefox/addon/to-google-translate/
-# "To Google Translate" is a brand name
-features-translate-get-to-google-translate = Get To { -brand-name-google } Translate
-
-# Obsolete string
-features-translate-get-the-extension = Get the Extension
-features-translate-once-installed-simply = Once installed, simply highlight the text you want to translate and right-click to pull up a menu with two options: 1) Takes to you translate.google.com with your selected text automatically entered into the translation field; or 2) Listen to audio pronunciation of the phrase (powered by { -brand-name-google } Text-to-Speech), which is helpful if you’re trying to learn a new language.
-features-translate-switch-languages-in = Switch Languages in { -brand-name-firefox }
-
-# $attrs (string) - link to https://support.mozilla.org/kb/use-firefox-another-language
-features-translate-if-you-are = If you are already using { -brand-name-firefox }, you can change your browser’s language or add languages to the { -brand-name-firefox } interface. <a { $attrs }>Learn how here.</a>
-
-# $attrs (string) - link to https://translate.google.com/about/languages/
-features-translate-with-the-to = *With the To { -brand-name-google } Translate extension, you can currently translate <a { $attrs }>these languages</a>:
-
-# $attrs (string) - link to https://translate.google.com/about/languages/
-features-translate-and-more = <a { $attrs }> and more!</a>
-
-features-translate-for-everyone-v2 = { -brand-name-firefox } is for everyone
-
-# Outdated string
-features-translate-for-everyone = Firefox is for everyone
+# HTML page title
+features-translate-translate-a-webpage-with-firefox = Translate a webpage with { -brand-name-firefox }
+features-translate-translate-the-web = Translate the web
+
+# HTML page description
+features-translate-firefox-translations-is-an-added = Translations is an added built-in translation feature that allows you to easily browse the web in your preferred language. Learn more about how this feature in { -brand-name-firefox } works, and how { -brand-name-mozilla } helps keep what you translate private.
```

Comment:
Should we change `Translations` to `{ -brand-name-firefox } Translations` to be more precise?